python/cudf/CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ project(
   VERSION "${RAPIDS_VERSION}"
   LANGUAGES CXX CUDA
 )
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
I wonder what CXX_STANDARD was it using before?

Reply:
No idea what the default is. Probably 17.